.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: ci
```

This looks cleaner in the GitHub UI with capitals, we should also use a more descriptive name.
```suggestion
name: Test & Lint
```